libjcat/jcat-blob.h

```diff
@@ -18,6 +18,7 @@ G_DECLARE_DERIVABLE_TYPE(JcatBlob, jcat_blob, JCAT, BLOB, GObject)
  * @JCAT_BLOB_KIND_SHA256:		SHA-256 checksum
  * @JCAT_BLOB_KIND_GPG:			GPG detached signature
  * @JCAT_BLOB_KIND_PKCS7:		PKCS-7 detached signature
+ * @JCAT_BLOB_KIND_SHA1:		SHA-1 checksum
```

Comment:
Since this isn't yet released, maybe group it with SHA256?

Reply:
I also pondered that, but then we might have a PKCS-8 in the future, so...